code/modules/martial_arts/martial.dm

```diff
@@ -7,6 +7,8 @@
 	var/datum/martial_art/base = null // The permanent style
 	var/deflection_chance = 0 //Chance to deflect projectiles
 	var/help_verb = null
+	var/can_use_guns = 1	//set to zero to prevent users of this style from using guns (sleeping carp, highlander). They can still pick them up, but not fire them.
```

Comment:
As this entire system is from TG, and as they have the same functionality, having the same var names is beneficial to both of us, in the long run.

Please make this the same as theirs: `var/no_guns = FALSE`